egs/ksponspeech/asr1/run.sh

```diff
@@ -221,9 +221,7 @@ fi
 if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
     echo "stage 5: Decoding"
     if [[ $(get_yaml.py ${train_config} model-module) = *transformer* ]] || \
-           [[ $(get_yaml.py ${train_config} model-module) = *conformer* ]] || \
```

Comment:
Could you recover these parts?
It is recently added.

Reply:
I have recovered these parts. Thank you!